main/src/ca/uwaterloo/flix/language/phase/jvm/GenFunAndClosureClasses.scala

```diff
@@ -146,10 +146,13 @@ object GenFunAndClosureClasses {
     // Methods
```

Comment:
The doc of genCode was probably outdated before you started changing things, but could you update it to reflect your new changes? both to understand the PR and for posterity